packages/turbo/android/src/main/java/com/reactnativeturbowebview/RNVisitableViewManager.kt

```diff
@@ -13,7 +13,9 @@ enum class RNVisitableViewEvent(val jsCallbackName: String) {
   MESSAGE("onMessage"),
   OPEN_EXTERNAL_URL("onOpenExternalUrl"),
   ON_ALERT("onWebAlert"),
-  ON_CONFIRM("onWebConfirm")
+  ON_CONFIRM("onWebConfirm"),
+  FORM_SUBMISSION_START("onFormSubmissionStarted"),
+  FORM_SUBMISSION_FINISH("onFormSubmissionFinished")
```

Comment:
Nitpicking, but there's some inconsistencies in how we name these (`ON_..` vs no prefix and past tense etc).
The inconsistency is there already though. WDYT?

Reply:
Yeah, let's fix that. Because we use this enum in a `sendEvent` let's just remove "on" prefix and keep the enum names synced with corresponding js callback name.